indigo/indigo-extras/src/main/scala/indigoextras/pathfinding/SearchGrid.scala

```diff
@@ -6,8 +6,11 @@ import indigoextras.pathfinding.GridSquare.EndSquare
 import indigoextras.pathfinding.GridSquare.ImpassableSquare
 import indigoextras.pathfinding.GridSquare.StartSquare
 
+import scala.annotation.nowarn
 import scala.annotation.tailrec
 
+@deprecated("Use the new indigoextras.pathfinding.PathFinder", "0.15.3")
+@nowarn("cat=deprecation")
 final case class SearchGrid(
```

Comment:
It's tempting to just remove this stuff. I doubt many people are using it. :thinking:

Reply:
I did not remove it.